src/stream.js

```diff
@@ -48,13 +49,17 @@ export const stream = (url, options) => {
   emitter.on('start', async () => {
     try {
       const fetch = await fetcher;
-      const response = await fetch(url, Object.assign({ credentials: 'omit' }, options));
+      const response = await fetch(
```

Comment:
Same reason as before. The try catch might not be needed.

Reply:
Needed for network errors.